patches/master_patch.patch

```diff
@@ -831,6 +832,18 @@ index 7e05c4e04f8e61b418d373a73bd8901347df4c76..e7e6d9fa5e0d54c7755fe3a38eca6f70
  
    // Holds strings needed by the external importer because the resource
    // bundle isn't available to the external process.
+diff --git a/chrome/utility/importer/firefox_importer.cc b/chrome/utility/importer/firefox_importer.cc
+index 7f8d598bca0243e62f11d5d2ce50d8c25ba88764..d198d90e8872eb1d7a371aeaca6119c73a6cda1c 100644
+--- a/chrome/utility/importer/firefox_importer.cc
++++ b/chrome/utility/importer/firefox_importer.cc
+@@ -147,7 +147,6 @@ void FirefoxImporter::StartImport(const importer::SourceProfile& source_profile,
+     ImportAutofillFormData();
+     bridge_->NotifyItemEnded(importer::AUTOFILL_FORM_DATA);
+   }
+-  bridge_->NotifyEnded();
```

Comment:
It is removed intentionally or it will be called twice in our subclass
https://github.com/brave/muon/pull/197/files#diff-355b9c92098e4fd3129c89c9f3c8a044R41